lib/dockerregistry/inventory.go

```diff
@@ -150,13 +151,12 @@ func ParseThinManifestFromFile(filePath string) (Manifest, error) {
 		return empty, err
 	}
 
-	imagesPath := thinManifest.ImagesPath
-	// Handle a relative path for "ImagesPath", if necessary.
-	if strings.HasPrefix(thinManifest.ImagesPath, ".") {
-		imagesPath = filepath.Join(
-			filepath.Dir(filePath),
-			thinManifest.ImagesPath)
-	}
+	// Get directory name holding this thin manifest.
+	subProject := filepath.Base(filepath.Dir(filePath))
+	imagesPath := filepath.Join(filepath.Dir(filePath),
+		"../../images",
```

Comment:
Nesting images under the manifest _might_ be less confusing, and _might_ still satisfy the OWNERs rules; but if this is what we're doing today then we should stick to it, I guess!

Reply:
My thoughts exactly. I can't find a strong enough reason to change it. Maybe that change can come in the future if we deem it necessary.